src/manifest.json

```diff
@@ -20,11 +20,20 @@
   "content_scripts": [{
     "matches": [
       "https://*.ghosterysearch.com/",
-      "http://localhost/*"
+      "https://*.ghosterysearch.com/?*",
```

Comment:
Does this match only if there is a query string, or did it previously not match when a query string was present? Or does this change do something else?